ansible/roles/ocp-workload-integreatly/tasks/workload.yml

```diff
@@ -9,7 +9,12 @@
 - name: Run Integreatly installer
   shell: |
           ansible-playbook -i "{{ inventory_hosts_file }}" \
-          playbooks/install.yml -e eval_self_signed_certs="{{ self_signed_certs_enabled }} -e rhsso_identity_provider_ca_cert_path={{ rhsso_identity_provider_ca_cert_path }} -e amq_streams={{ amq_streams }}"
+          playbooks/install.yml \
+          -e eval_self_signed_certs="{{ self_signed_certs_enabled }} \
+          -e rhsso_identity_provider_ca_cert_path={{ rhsso_identity_provider_ca_cert_path }} \
+          -e amq_streams={{ amq_streams }} -e rhsso_seed_users_password={{ evals_password }} \
+          -e rhsso_evals_admin_password={{ admin_password }} \
+          -e rhsso_cluster_admin_password={{  cluster_admin_password }}"
```

Comment:
I don't think this is needed? We're not overriding the cluster_admin user. Should this not be customer_admin?